JASP-Engine/JASP/R/LSbinomialestimation.R

```diff
@@ -956,8 +956,8 @@ LSbinomialestimation   <- function(jaspResults, dataset, options, state = NULL){
                                                 options[["plotsIterativeBF"]])
           }
           
-          if (all(is.na(tempCIPP[1:2]))){
-            tempInt <- "∅"
+          if (all(is.na(temp_CIPP[1:2]))){
```

Comment:
Ah, here's another one, should be `tempCIPP`

Reply:
Maybe I should just copy the LSbinomialestimation from stable?